Control/TMSITable.cpp

```diff
@@ -819,13 +819,21 @@ vector< vector<string> > TMSITable::tmsiTabView(int verbosity, bool rawFlag, uns
 				}
 			} else if (header == "CREATED" || header == "ACCESSED") {
 				// Print seconds as a time value.
-				row.push_back(prettyAge(now - query.getResultInt(col)));
+				if (rawFlag) {
+					row.push_back(format("%d", query.getResultInt(col)));
```

Comment:
Ah, interesting point. Your call what the semantics should be, but
regardless let's update the documentation to state that explicitly.

On Wed, Jun 17, 2015 at 6:25 PM, Omar Ramadan <notifications@github.com>
wrote:

> In Control/TMSITable.cpp
> <https://github.com/endaga/openbts/pull/19#discussion_r32692645>:
>
> > @@ -819,13 +819,21 @@ vector< vector<string> > TMSITable::tmsiTabView(int verbosity, bool rawFlag, uns
> >  				}
> >  			} else if (header == "CREATED" || header == "ACCESSED") {
> >  				// Print seconds as a time value.
> > -				row.push_back(prettyAge(now - query.getResultInt(col)));
> > +				if (rawFlag) {
> > +					row.push_back(format("%d", query.getResultInt(col)));
>
> It depends on how you interpret what the raw flag should do. The
> description says it's spitting out DB values, but I think giving the deltas
> is acceptable and would eliminate possible issues with unsynchronized
> clocks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/endaga/openbts/pull/19/files#r32692645>.
>